content/post/099-identity-ipfs-ion.md

```diff
@@ -0,0 +1,59 @@
+---
+date: 2020-06-11
+url: /2020-06-11-identity-ipfs-ion/
+tags: [ipfs, identity]
+header_image: blog-header-ion.png
+title: Decentralized Identity, IPFS and ION
+author: Dietrich Ayala
+---
+
+Identity is a key part of how we all interact with each other on the internet every day—sometimes every moment of every day. And sometimes each raging moment as we do the “forgot your password?” dance yet again. Regardless, forging a path forward for user-controlled online identity is a table-stakes requirement for the dweb to succeed.
+
+But, identity is hard. There are no easy solutions for interoperable decentralized digital assertions of self. Privacy, security, validity, access control, surveillance, GDPR, KYC, anonymity. It’s like Inception but each nested dream level is the same minefield, and those minefields are littered with projects that’ve tilted at the identity windmill, leaving naught behind but vinyl stickers slowly being covered up on our laptops.
+
+Hope has not forsaken this land. The [Decentralized Identity Foundation](https://identity.foundation/) has been plugging away at this challenge for many years, as have groups like [Rebooting the Web of Trust](https://www.weboftrust.info/). And there’ve been efforts to build decentralized identity systems on top of IPFS, such as [IPID](https://github.com/johnnycrunch/ipid), [Nomios.io](https://nomios.io/), [the IPFS IDM](https://github.com/ipfs-shipyard/pm-idm), and most recently the [Ceramic Network](https://www.ceramic.network/).
+
+However, today we’re celebrating the launch of a decentralized identity protocol and service from an unlikely place: Microsoft.
+
+Microsoft has been increasingly present in open-source tools and services in the past few years, and has now [launched a standards-based decentralized identity service called Ion](https://techcommunity.microsoft.com/t5/identity-standards-blog/ion-booting-up-the-network/ba-p/1441552).
```

Comment:
Their announcement post calls it ION, but we refer to it as Ion in a few places, so I just want to make sure we're being consistent with the proper name.

Reply:
Yeah it's ION, thanks for catching the rogue Ions